internals/webpack/webpack.base.babel.js

```diff
@@ -28,6 +28,8 @@ const buildSpritePlugin = (name) => new SpritesmithPlugin({
   },
 });
 
+const isProduction = process.env.NODE_ENV === 'production';
+
 module.exports = (options) => {
```

Comment:
Can you see the options object here? It's passed from more specific files – dev, prod, etc.

Instead of adding another check for NODE_ENV, add a `styleHMR` property in `webpack.prod.babel.js` and just pass it to style-loader

Reply:
Aaach, I forgot about it. Good point! :P